frontend/javascripts/dashboard/explorative_annotations_view.tsx

```diff
@@ -545,6 +560,9 @@ class ExplorativeAnnotationsView extends React.PureComponent<Props, State> {
         scroll={{
           x: "max-content",
         }}
+        onChange={(_pagination, _filters, _sorter, extra) => {
+          this.setState({ filteredAnnotations: extra.currentDataSource });
```

Comment:
I think `filteredAnnotations` needs to be initialized when rendering the table, since this line will only be triggered onChange. The current symptome is that when pressing "Archive All" as the first action after loading the page, there is a toast stating "No annotations available to archive.". After filtering once, the archiving works as expected (searching doesn't make it work, only changing the Owner & Teams filter).